public/test.html

```diff
@@ -0,0 +1,13 @@
+<html>
+
+<head>
+  <link href="style.css" rel="stylesheet" type="text/css" />
```

Comment:
With my component I made the css modular as in "style.module.css" I'm not sure if you need to do this or if it will work the way you have it on your proxy, just something to keep in mind